src/ui/EditorNS/editor.cpp

```diff
@@ -125,6 +134,20 @@ namespace EditorNS
         return out;
     }
 
+    void Editor::on_loadFinished()
+    {
+        updateBackground();
+    }
+
+    void Editor::updateBackground()
+    {
+        QString js = "rgb2hex($('.cm-s-'+editor.getOption('theme')).css('background-color'));";
+        m_webView->page()->runJavaScript(js, [&] (const QVariant& v){
```

Comment:
I'd prefer asking javascript to give us the background color by calling an already defined js function, instead of sending that string which is out of context.